### PR TITLE
doc/cephfs shell: Add usage of quota, df and du commands

### PR DESCRIPTION
**Pull request by varshar16**
Fixes: https://tracker.ceph.com/issues/44788
Signed-off-by: Varsha Rao <varao@redhat.com>

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by varshar16** [comment]
jenkins retest this please

**Comment by varshar16** [comment]
jenkins render docs

**Comment by varshar16** [comment]
https://docs.ceph.com/ceph-prs/34948/cephfs/cephfs-shell/

File: doc/cephfs/cephfs-shell.rst [hunk 1]
```diff
@@ -441,3 +441,51 @@ Usage :
...
+Show disk usage of a directory
+
+Usage :
+
+    du [-h] [-r] [paths [paths ...]]
+
+* paths - name of the directory
+
+Options:
+  -h, --help   Shows the help message
```
Comment (rishabh-d-dave) [inline doc/cephfs/cephfs-shell.rst:471]: An option got missed - `  -r          Recursive Disk usage of all directories.`.

Comment (varshar16) [reply doc/cephfs/cephfs-shell.rst:471]: Done.

**Comment by varshar16** [comment]
jenkins render docs

**Pull request merged**